### ISSUE TITLE
Uninstall DSpellCheck

### ISSUE THREAD

**Issue by ygoe** [notepad-plus-plus/notepad-plus-plus#4083]
### Description of the Issue
DSpellCheck is a new plugin that was installed with a recent NPP version. I don't need it and always disable the checking. Now I've divcovered that it hijacks the very important hotkey Alt+D which opens the File menu in the German language.

There used to be a plugin manager but that seems to have gone away, too. So there's no way to uninstall the spell checking and restore the correct function of the Alt+D hotkey.

### Steps to Reproduce the Issue
1. Install Notepad++ with German translation.
2. Press Alt+D to open the File menu ("Datei")

### Expected Behavior
File menu opens

### Actual Behavior
Some other unknown context menu opens at the mouse cursor

### Debug Information
Notepad++ v7.5.4   (32-bit)
Build time : Jan  1 2018 - 01:47:45
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 10 (64-bit)
Plugins : DSpellCheck.dll mimeTools.dll NppConverter.dll NppExport.dll

**Comment by SinghRajenM** [comment]
> So there's no way to uninstall the spell checking

Option is given at the time of installation. You can uncheck DSpellCheck during installation, then it will not be installed.

**Comment by ygoe** [comment]
Okay, so how can I *uninstall* it? Running the installer again and deselect the option doesn't help. What file do I have to delete?

**Comment by sasumner** [comment]
Remove the offending plugin's DLL file from the plugins directory; this will "uninstall" it.

**Comment by Watilin** [comment]
Would be great to have the possibility to uninstall it from the plugin manager anyway— just like any other plugin.

With a French translation, spell checking hijacks the `Alt+A` shortcut which used to open the Display (*Affichage*) menu. Not as important as File, but still.

**Comment by woenho** [comment]
remove folder ->
C:\Program Files\Notepad++\plugins\DSpellCheck

**Comment by SinghRajenM** [comment]
>Okay, so how can I uninstall it? Running the installer again and deselect the option doesn't help. What file do I have to delete?

@ygoe , PR has been submitted which will work exactly as you stated (Running the installer again and deselect the option). Let's hope it will get merged.

**Comment by woenho** [comment]
hum.... I do not know how uncheck option at install time. but It is not need uninstall process, only remove that plugin. so remove that,====> delete     C:\Program Files (x86)\Notepad++\plugins\DSpellCheck
okey?

**Comment by ledents** [comment]
DSpellCheck makes NPP crash on exit.  Since some time i discovered that when exiting notepad++ I get a system crash dialog, telling me the application has stopped working. Even a fresh reinstall of NPP v7.5.8 (64bit) didn't solve the issue. When I tried to run it without any plugins it was working fine. After several uninstallation and reinstallations with different selection of the 3 plugin choices at NPP installation, I figured that DSpellCheck was the culprid for me.
